hack/deploy.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
+	OPERATOR_IMAGE="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:cincinnati-operator"
+	GRAPH_DATA_IMAGE="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:cincinnati-graph-data-container"
+
+	echo "Openshift CI detected, deploying using image $OPERATOR_IMAGE and ${GRAPH_DATA_IMAGE}"
+
+	sed -i "s|quay.io/cincinnati/cincinnati-operator:latest|$OPERATOR_IMAGE|" deploy/operator.yaml
+	sed -i "s|quay.io/cincinnati/cincinnati:latest|quay.io/app-sre/cincinnati:2873c6b|" deploy/operator.yaml
+	sed -i "s|your-registry/your-repo/your-init-container|$GRAPH_DATA_IMAGE|" deploy/crds/cincinnati.openshift.io_v1beta1_cincinnati_cr.yaml
+
+else
+	if ! [ -n "$KUBECONFIG" ]; then
+		echo "KUBECONFIG environment variable must be set."
+		exit 1
+	fi
+	if ! [ -n "$GRAPH_DATA_IMAGE" ]; then
+		echo "GRAPH_DATA_IMAGE environment variable must be set."
+		exit 1
+	fi
+	sed -i "s|quay.io/cincinnati/cincinnati:latest|quay.io/app-sre/cincinnati:2873c6b|" deploy/operator.yaml
+	sed -i "s|your-registry/your-repo/your-init-container|$GRAPH_DATA_IMAGE|" deploy/crds/cincinnati.openshift.io_v1beta1_cincinnati_cr.yaml
```

Comment:
Also consider making OPERATOR_IMAGE a variable that can be overridden. And since the sed statements are the same for both if and else,you could move those outside the if-else.

Reply:
Done